lib/fluent/plugin/out_kafka_buffered.rb

```diff
@@ -62,6 +62,11 @@ class Fluent::KafkaOutputBuffered < Fluent::BufferedOutput
 The codec the producer uses to compress messages.
 Supported codecs: (gzip|snappy)
 DESC
+  config_param :max_buffer_size, :integer, :default => nil,
```

Comment:
These parameters are not needed because out_kafka_buffered and out_kafka2 don't use original `produce` method.